### PR TITLE
settings

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/sql/settings.md [hunk 1]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+        }
+      }
+    }
+  }
+}
+```
+
+### Example 2
+
+After the setting is updated, query result is like:
```
Comment (aetter) [inline docs/sql/settings.md:346]: "After you update the setting, queries return as JSON."

File: docs/sql/settings.md [hunk 2]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+1.  The default value is false.
+2.  This setting is node scope.
+3.  This setting can be updated dynamically.
+
+### Example
+
+You can update the setting with a new value like this.
+
+SQL query:
```
Comment (aetter) [inline docs/sql/settings.md:422]: Feels unnecessary given that it's the same exact call as earlier with a slightly different payload.

File: docs/sql/settings.md [hunk 3]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.cursor.enabled" : "true"
+  }
+}'
+```
+
+Result set:
```
Comment (aetter) [inline docs/sql/settings.md:432]: Result unnecessary.

File: docs/sql/settings.md [hunk 4]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.query.response.format" : "json"
+  }
+}'
+```
+
+Result set:
```
Comment (aetter) [inline docs/sql/settings.md:324]: Result unnecessary.

File: docs/sql/settings.md [hunk 5]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+You can set the default `fetch_size` for all queries that are supported by pagination.
+An explicit `fetch_size` passed in request overrides this value.
+
+1.  The default value is 1000.
+2.  This setting is node scope.
+3.  This setting can be updated dynamically.
+
+### Example
+
+You can update the setting with a new value like this.
```
Comment (aetter) [inline docs/sql/settings.md:463]: Ditto

File: docs/sql/settings.md [hunk 6]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.cursor.fetch_size" : "50"
+  }
+}'
+```
+
+Result set:
```
Comment (aetter) [inline docs/sql/settings.md:475]: Ditto

File: docs/sql/settings.md [hunk 7]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+1.  The default value is 1 minute.
+2.  This setting is node scope.
+3.  This setting can be updated dynamically.
+
+### Example
+
+You can update the setting with a new value like this.
+
+SQL query:
```
Comment (aetter) [inline docs/sql/settings.md:508]: Ditto

File: docs/sql/settings.md [hunk 8]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.cursor.keep_alive" : "5m"
+  }
+}'
+```
+
+Result set:
```
Comment (aetter) [inline docs/sql/settings.md:518]: Ditto

File: docs/sql/settings.md [hunk 9]
```diff
@@ -295,3 +295,240 @@ Result set:
   }
 }
 ```
+
+## opendistro.sql.query.response.format
+
+### Description
+
+You can set default response format of the query. The supported formats include: jdbc, json, csv, raw, and table.
```
[the remark this answers is not in the document]
Comment (ashwinkumar12345) [reply docs/sql/settings.md:303]: Done.

File: docs/sql/settings.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/sql/settings.md:346]: Done.

File: docs/sql/settings.md [hunk 10]
```diff
@@ -295,3 +295,240 @@ Result set:
...
+
+
+### Example 1
+
+You can update the setting with a new value like this.
+
+SQL query:
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
```
[the remark this answers is not in the document]
Comment (ashwinkumar12345) [reply docs/sql/settings.md:317]: Added to all queries.

File: docs/sql/settings.md (hunk 8 above)
Comment (ashwinkumar12345) [reply docs/sql/settings.md:518]: Done.

File: docs/sql/settings.md (hunk 7 above)
Comment (ashwinkumar12345) [reply docs/sql/settings.md:508]: Done.

File: docs/sql/settings.md (hunk 6 above)
Comment (ashwinkumar12345) [reply docs/sql/settings.md:475]: Done.

**Pull request closed without merging**